lib/all-badge-examples.spec.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const { expect } = require('chai');
+
+const allBadgeExamples = require('./all-badge-examples');
+
+describe('The badge examples', function () {
```

Comment:
Do you think its worth having another test here which just iterates over all examples (rather than just the appveyor ones) and runs `prepareExample()` on each one. It doesn't have to assert anything other than an error wasn't thrown. That way we'll get a failure on the server tests if an invalid example (without a `previewUri`) is added in a PR.

Reply:
one step ahead of me :)